docs/appendices/release-notes/unreleased.rst

```diff
@@ -77,24 +77,6 @@ Fixes
   :ref:`cursors <sql-fetch>`, even if the ``CURSOR`` was explicitly or
   automatically (session terminated) closed.
 
-- Fixed an issue that caused the returned column names to be missing the
```

Comment:
in 4.8.4 we kept entries until after 5.0.2 -> think need to keep those here until after 5.2.1?
https://github.com/crate/crate/pull/13044#discussion_r972070378